# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../metamail/40_mm-2.7-sml.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- mm2.7/src/metamail/splitmail.c.sml	Fri Jul 30 15:52:01 1999
+++ mm2.7/src/metamail/splitmail.c	Fri Jul 30 15:52:14 1999
@@ -42,8 +42,8 @@
 #define VERBOSEDELIVERYCMD VerboseDeliveryCmd
 #else
 extern char *getenv();
-#define NORMALDELIVERYCMD "/usr/lib/sendmail -t -oi"
-#define VERBOSEDELIVERYCMD "/usr/lib/sendmail -t -v -oi"
+#define NORMALDELIVERYCMD "/usr/sbin/sendmail -t -oi"
+#define VERBOSEDELIVERYCMD "/usr/sbin/sendmail -t -v -oi"
 #endif
 
 usageexit() {
--- mm2.7/src/bin/audiosend.sml	Fri Jul 30 15:49:33 1999
+++ mm2.7/src/bin/audiosend	Fri Jul 30 15:49:44 1999
@@ -110,7 +110,7 @@
     switch ("$which")
         case 1:
           echo -n "Sending mail, please wait...  "
-          /usr/lib/sendmail "$to" "$cc" < $fname
+          /usr/sbin/sendmail "$to" "$cc" < $fname
           if (! $status) then
             echo "Done."
             rm $fname
--- mm2.7/src/bin/mailserver.sml	Fri Jul 30 15:50:19 1999
+++ mm2.7/src/bin/mailserver	Fri Jul 30 15:50:36 1999
@@ -76,7 +76,7 @@
 probably run in some circumstance other than mail delivery.
 --------------------
 !
-	cat $TmpFile - | /usr/lib/sendmail "$MAINTAINER"
+	cat $TmpFile - | /usr/sbin/sendmail "$MAINTAINER"
 	# Takes the rest of the message from standard input
 	rm $TmpFile
 	exit 0
@@ -94,7 +94,7 @@
 
 The file you requested, if it exists, will not be sent to you.
 !
-	/usr/lib/sendmail -t < $TmpFile
+	/usr/sbin/sendmail -t < $TmpFile
 	rm $TmpFile
 	exit 0
 endif
@@ -116,7 +116,7 @@
 	echo "--------------------------------" >> $TmpFile
 	ls -R >> $TmpFile
 	echo "" >> $TmpFile
-	/usr/lib/sendmail -t < $TmpFile
+	/usr/sbin/sendmail -t < $TmpFile
 	rm $TmpFile
 	exit 0
 endif
@@ -144,7 +144,7 @@
 	echo "Command failed:" >> $TmpFile
 	echo "  metasend -b -t $FromName -f $Subject -m $ct -s Re: $Subject" >> $TmpFile
 	echo "" >> $TmpFile
-	/usr/lib/sendmail -t < $TmpFile
+	/usr/sbin/sendmail -t < $TmpFile
 	rm $TmpFile
 	exit 0
 endif
@@ -157,7 +157,7 @@
 	echo The file: "$Subject" >> $TmpFile
 	echo was sent to: "$FromName" >> $TmpFile
 	echo "" >> $TmpFile
-	/usr/lib/sendmail -t < $TmpFile
+	/usr/sbin/sendmail -t < $TmpFile
 endif
 
 rm -f $TmpFile
--- mm2.7/src/man/mailto.1.sml	Fri Jul 30 15:51:30 1999
+++ mm2.7/src/man/mailto.1	Fri Jul 30 15:51:37 1999
@@ -230,7 +230,7 @@
    dot -- controls whether or not a period alone on a line 
         should be interpreted as terminating your mail
    ignore -- controls whether or not interrupts are ignored
-   verbose -- controls the verbosity of output from /usr/lib/sendmail
+   verbose -- controls the verbosity of output from /usr/sbin/sendmail
    quiet -- controls the verbosity of output from the mailto program.
    keepblind -- controls whether or not a 'blind' copy of the mail is kept.
   commasonly -- controls whether or not a space character
